### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by amaredeus**
## 2.3.0-RELEASE (2021-12-07)

1. Enable poap scheduling (POAP API integration)
2. Remove Datadog
3. Enable poap config via channel flow
4. Add poap config status command to display list of authorized users and roles
5. Rename `/poap schedule` -> `/poap mint`
6. Enable POAP distribution
   - reduce poap claim deleted to 5 minutes
   - display poap claimed message
   - fix poap duration for auto end events
7. Clear expired POAPs on startup
   - surround error log in try/catch block
8. Add a security check for the poap urls that are passed
   - default platform to discord

### CODE REVIEW

**Review by austinmitchem — APPROVED** [review]

File: src/app/service/poap/config/ModifyPOAP.ts [hunk 1]
```diff
@@ -1,34 +1,46 @@
...
+	MessageReaction,
+	Role,
+	TextChannel,
+} from 'discord.js';
+import ValidationError from '../../../errors/ValidationError';
 import { Collection } from '@discordjs/collection';
 import {
 	BulkWriteError,
-	Collection as MongoCollection,
+	Collection as MongoCollection, CollectionInsertManyOptions,
```
Comment (nonsensecodes) [inline src/app/service/poap/config/ModifyPOAP.ts:14]: Will you put `CollectionInsertManyOptions` on its own line?

**Review by nonsensecodes — CHANGES_REQUESTED** [review]
Just one change. Otherwise, everything looks fantastic!

File: src/app/service/poap/config/ModifyPOAP.ts (hunk 1 above)
Comment (amaredeus) [reply src/app/service/poap/config/ModifyPOAP.ts:14]: Sure but FYI I'll need to create a new PR since this is the 2nd stage of the review

**Comment by amaredeus** [comment]
Merging as this is previously approved

**Pull request merged**